### PR TITLE
Fix "whitespace" words - docs/dotnet-api-docs

### PR DESCRIPTION
**Pull request by lucasnobredev**
## Fix "whitespace" words - docs/dotnet-api-docs.

This PR came out from this [issue](https://github.com/dotnet/docs/issues/4439) on docs/dotnet-api-docs. It's all about docs/dotnet-api-docs.

## Brief explanation
We should use **white-space** for adjectives and **white space** for nouns.

## Suggested Reviewers
@mairaw @rpetrusha @wstaelens

### CODE REVIEW

File: xml/System.Security.Cryptography.Xml/XmlDsigExcC14NTransform.xml [hunk 1]
```diff
@@ -136,7 +136,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The `inclusiveNamespacesPrefixList` parameter contains a whitespace-delimited list of namespace prefixes to canonicalize using the standard canonicalization algorithm rather than the exclusive canonicalization algorithm.  To specify the default namespace, which does not have a prefix, pass the special prefix "#default".  
+ The `inclusiveNamespacesPrefixList` parameter contains a white space delimited list of namespace prefixes to canonicalize using the standard canonicalization algorithm rather than the exclusive canonicalization algorithm.  To specify the default namespace, which does not have a prefix, pass the special prefix "#default".
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography.Xml/XmlDsigExcC14NTransform.xml]: white space delimited --> white-space delimited

File: xml/System.Security.Cryptography.Xml/XmlDsigExcC14NTransform.xml [hunk 2]
```diff
@@ -172,7 +172,7 @@
...
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography.Xml/XmlDsigExcC14NTransform.xml]: white space delimited --> white-space delimited

File: xml/System.Security.Cryptography.Xml/XmlDsigExcC14NTransform.xml [hunk 3]
```diff
@@ -341,7 +341,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Security.Cryptography.Xml.XmlDsigExcC14NTransform.InclusiveNamespacesPrefixList%2A> property contains a whitespace-delimited list of namespace prefixes to canonicalize using the standard canonicalization algorithm rather than the exclusive canonicalization algorithm.  To specify the default namespace, which does not have a prefix, pass the special prefix "#default".  
+ The <xref:System.Security.Cryptography.Xml.XmlDsigExcC14NTransform.InclusiveNamespacesPrefixList%2A> property contains a white space delimited list of namespace prefixes to canonicalize using the standard canonicalization algorithm rather than the exclusive canonicalization algorithm.  To specify the default namespace, which does not have a prefix, pass the special prefix "#default".
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography.Xml/XmlDsigExcC14NTransform.xml]: white space delimited --> white-space delimited

File: xml/System.Security.Cryptography.Xml/XmlDsigExcC14NWithCommentsTransform.xml [hunk 4]
```diff
@@ -105,7 +105,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The `inclusiveNamespacesPrefixList` parameter contains a whitespace-delimited list of namespace prefixes to canonicalize using the standard canonicalization algorithm rather than the exclusive canonicalization algorithm.  To specify the default namespace, which does not have a prefix, pass the special prefix "#default".  
+ The `inclusiveNamespacesPrefixList` parameter contains a white space delimited list of namespace prefixes to canonicalize using the standard canonicalization algorithm rather than the exclusive canonicalization algorithm.  To specify the default namespace, which does not have a prefix, pass the special prefix "#default".
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography.Xml/XmlDsigExcC14NWithCommentsTransform.xml]: white space delimited --> white-space delimited

File: xml/System.Windows.Documents/FlowDocument.xml [hunk 5]
```diff
@@ -855,7 +855,7 @@
  A string specifying a font family name.  For example, `"Arial"` or `"Century Gothic"`.  
   
  *fontFamilyNamesList*  
- A string specifying multiple font family names, each separated by a comma (any whitespace following a comma is ignored).  The first font family specified serves as the primary font family; subsequent font families serve as fallback families to be used in cases where the primary font family is unavailable or not applicable.  For example, `"Arial, Century Gothic"` specifies Arial as the primary font family, with Century Gothic as the fallback font family.  
+ A string specifying multiple font family names, each separated by a comma (any white-space following a comma is ignored).  The first font family specified serves as the primary font family; subsequent font families serve as fallback families to be used in cases where the primary font family is unavailable or not applicable.  For example, `"Arial, Century Gothic"` specifies Arial as the primary font family, with Century Gothic as the fallback font family.
```
Comment (rpetrusha) [inline xml/System.Windows.Documents/FlowDocument.xml]: white-space --> white space

File: xml/System.Windows.Documents/TextElement.xml [hunk 6]
```diff
@@ -283,7 +283,7 @@
  A string specifying a font family name.  For example, `"Arial"` or `"Century Gothic"`.  
   
  *fontFamilyNamesList*  
- A string specifying multiple font family names, each separated by a comma (any whitespace following a comma is ignored).  The first font family specified serves as the primary font family; subsequent font families serve as fallback families to be used in cases where the primary font family is unavailable or not applicable.  For example, `"Arial, Century Gothic"` specifies Arial as the primary font family, with Century Gothic as the fallback font family.  
+ A string specifying multiple font family names, each separated by a comma (any white-space following a comma is ignored).  The first font family specified serves as the primary font family; subsequent font families serve as fallback families to be used in cases where the primary font family is unavailable or not applicable.  For example, `"Arial, Century Gothic"` specifies Arial as the primary font family, with Century Gothic as the fallback font family.
```
Comment (rpetrusha) [inline xml/System.Windows.Documents/TextElement.xml]: white-space --> white space

File: xml/System.Windows.Media/FontFamily.xml [hunk 7]
```diff
@@ -132,7 +132,7 @@ Example of typefaces that are members of the "Times New Roman" font family
  A string specifying a font family name.  For example, `"Arial"` or `"Century Gothic"`.  
   
  *fontFamilyNamesList*  
- A string specifying multiple font family names, each separated by a comma (any whitespace following a comma is ignored).  The first font family specified serves as the primary font family; subsequent font families serve as fallback families to be used in cases where the primary font family is unavailable or not applicable.  For example, `"Arial, Century Gothic"` specifies Arial as the primary font family, with Century Gothic as the fallback font family.  
+ A string specifying multiple font family names, each separated by a comma (any white-space following a comma is ignored).  The first font family specified serves as the primary font family; subsequent font families serve as fallback families to be used in cases where the primary font family is unavailable or not applicable.  For example, `"Arial, Century Gothic"` specifies Arial as the primary font family, with Century Gothic as the fallback font family.
```
Comment (rpetrusha) [inline xml/System.Windows.Media/FontFamily.xml]: white-space --> white space

File: xml/System.Xaml/XamlDuplicateMemberException.xml [hunk 8]
```diff
@@ -19,7 +19,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- If you handle this exception from a XAML node stream, members might be reported as duplicates in several possible situations. For example, an explicit property element for a XAML content property might be preceded by whitespace that is interpreted as content, if the collection is whitespace significant. Or the XAML writer might be attempting to reset a property that should instead be using a `GetObject` default value. If you analyze individual nodes in the XAML node stream, you can often determine the source of the problem.  
+ If you handle this exception from a XAML node stream, members might be reported as duplicates in several possible situations. For example, an explicit property element for a XAML content property might be preceded by white space that is interpreted as content, if the collection is white space significant. Or the XAML writer might be attempting to reset a property that should instead be using a `GetObject` default value. If you analyze individual nodes in the XAML node stream, you can often determine the source of the problem.
```
Comment (rpetrusha) [inline xml/System.Xaml/XamlDuplicateMemberException.xml]: white space significant --> white-space significant

File: xml/System.Xaml/XamlXmlReaderSettings.xml [hunk 9]
```diff
@@ -175,7 +175,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The XML whitespace concepts are not robust enough to apply to XAML, and are typically ignored by XAML readers.  
+ The XML white space concepts are not robust enough to apply to XAML, and are typically ignored by XAML readers.
```
Comment (rpetrusha) [inline xml/System.Xaml/XamlXmlReaderSettings.xml]: white space concepts --> white-space concepts
also, unrelated to your PR, no comma after XAML

**Comment by lucasnobredev** [comment]
@rpetrusha  I'm sorry for the errors. I had some doubts about adjectives and nouns. But I'm learning day by day with you.
Done!

**Comment by rpetrusha** [comment]
There's no need to apologize, @lucasnobredev. We appreciate the work that you've been doing, and it's a pleasure to work on it with you.

**Pull request merged**